homeassistant/components/sentry/__init__.py

```diff
@@ -114,6 +148,8 @@ def process_before_send(
                     additional_tags[group2] = group3
             else:
                 # Not the "homeassistant" package, this third-party
+                if not options.get(CONF_EVENT_THIRD_PARTY_PACKAGES):
+                    return None
```

Comment:
Should we really not report any event if this option is disabled?

Reply:
Wait, I missed that we've already branched down off the main branch, right? So it's just in some cases this will be a case.